.gitignore

```diff
@@ -32,6 +32,8 @@
 .ensime_lucene
 .generated-mima*
 .vscode/
+.bloop/
```

Comment:
Please not introduce extraneous changes

Reply:
what's this for? vscode?

Reply:
This is for scala metals: https://github.com/scalameta/metals
I will remove it if you think it is not good to add them in this PR

Reply:
it's fine, but better in a separated PR

Reply:
Cleaned